### ISSUE TITLE
Linuxkit/Kubernetes & Raspberry Pi

### ISSUE THREAD

**Issue by HurricanKai** [linuxkit/kubernetes#90]
I've tried to get this to build for the past 3 days, and I don't know what else to do but ask here.
I've got this to build in a few minutes using `KUBE_FORMATS=iso-bios` and `KUBE_RUNTIME=cri-containerd`. After I tried to install this onto my PIs, which obviously did not work, i found [docs/platform-rpi3](https://github.com/linuxkit/linuxkit/blob/master/docs/platform-rpi3.md) which then did not build on my Windows machine (amd64 I think, i still haven't found out how to figure this out on Windows), this is where my problems startet.
Tried building on a PI, couldn't get linuxkit to build (from the tags/v0.7 branch, as well as the master branch)
I've setup a Ubuntu Machine using Hyper-V, from there i installed Docker and qemu, started a docker container from there, linked in the docker daemon (Docker-in-Docker told me to do this), started using qemu-aarch64-static (also tried qemu-arm-static, which gave me the same environment a Raspberry PI has too, `armv7` aka `armhf`) this have me an environment which was arm, but I could not get past building the master .tar because at that point things would go really wrong because of qemu some Fatal Error which i could find nothing about.
And here I am now. I'd be very greatful if anyone has some advice, or even something specific I can look up / try.

**Comment by HurricanKai** [comment]
It looks like I have gotten a step further by updating my PI to the experimental version of Ubuntu Mate, which makes use of the 64 bit processor, and is armv8, that way linuxkit did build, and i was able (after some tinkering, had to change some version) to build an rpi3 image, it does boot now (again, complications, for some reason win Fat32 formatting doesn't work) but memory isn't found. I hope to be able to fix this, but at least I got some image now.

**Comment by HurricanKai** [comment]
After two more days of trying to get this to work I decided to use my initial build and test if it works on my PC (16 GB RAM, note this) inside of HyperV.
Turns out, the Kube Master, not doing anything but having run kubeadm init & rebooting uses *8 GB* of Memory. Not sure why the rpi3 format even exsists, it clearly cannot ever be supported (I wouldnt consider this project to have that many containers) even the latest Pi, Pi3 B+ only has 1 GB of Memory.
